config/metrics-prometheus-collector/collector.role.yaml

```diff
@@ -6,3 +6,6 @@ rules:
 - apiGroups:  [ "", "apps", "batch", "quotamanagement.usagemetricscollector.sigs.k8s.io"]
   resources: [ "*" ]
   verbs: [ get, list, watch ]
+- apiGroups: ["coordination.k8s.io"]
+  resources: ["leases"]
+  verbs: [ "*" ]
```

Comment:
Can we constrain the verbs more? This should be sufficient:

```
verbs:
  - get
  - create
  - patch
  - update
  - delete
```

Reply:
Thanks, updated the verbs!